shardwidth/helper.go

```diff
@@ -6,6 +6,13 @@ import (
 	"math/bits"
 )
 
+// Exponent controls the size of each shard
+//
+// # Warnings
+// - changing this value WILL corrupt any data sets created with a different value **
```

Comment:
maybe `**changing this value...value**` so markdown will probably think it's bold?

Reply:
This formatting should work everywhere, and I'm hoping with a `Warnings` header, we won't need to bold the actual warning items